### PR TITLE
Lesson 23

### PR DESCRIPTION
**Pull request by daianequeiroz**
Finalized lesson 23 - JavaScript. @anton-shevchook @chriztie

### CODE REVIEW

File: Exercises/JS/06-js-logical-operators/js/script.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+let hour = Number(prompt ("Enter the current hour (out of 0-23):"));
+
+
+
+if (hour < 0 || hour > 23){
+    alert("Input is incorrect!");
+}else if (hour < 8 || hour >= 18){
+    alert("Outside of office hours!");
+}else if (hour >= 8 && hour < 18){
+    alert("We are open!")
```
Comment (anton-shevchook) [inline Exercises/JS/06-js-logical-operators/js/script.js]: Don't forget semicolon `;` 😉

File: Exercises/JS/06-js-logical-operators/js/script.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+let hour = Number(prompt ("Enter the current hour (out of 0-23):"));
+
+
+
```
Comment (anton-shevchook) [inline Exercises/JS/06-js-logical-operators/js/script.js]: Better not to do unnecessary line breaks.

File: Exercises/JS/06-js-logical-operators/js/script.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+let hour = Number(prompt ("Enter the current hour (out of 0-23):"));
```
Comment (anton-shevchook) [inline Exercises/JS/06-js-logical-operators/js/script.js:1]: Good that you converted to "number" type before comparing.

**Review by anton-shevchook — APPROVED** [review]
Good job!
You can merge this pull request.

File: Exercises/JS/06-js-logical-operators/js/script.js (hunk 1 above)
Comment (daianequeiroz) [reply Exercises/JS/06-js-logical-operators/js/script.js]: I included the semicolon.

File: Exercises/JS/06-js-logical-operators/js/script.js (hunk 2 above)
Comment (daianequeiroz) [reply Exercises/JS/06-js-logical-operators/js/script.js]: Removed.

File: Exercises/JS/06-js-logical-operators/js/script.js (hunk 3 above)
Comment (daianequeiroz) [reply Exercises/JS/06-js-logical-operators/js/script.js:1]: Thanks.

**Pull request merged**